jams/core.py

```diff
@@ -496,49 +498,21 @@ class Sandbox(JObject):
     pass
 
 
-class JamsFrame(pd.DataFrame):
-    '''A data-frame class for JAMS.
+Observation = namedtuple('Observation',
+                         ['time', 'duration', 'value', 'confidence'])
+'''Core observation type: (time, duration, value, confidence).'''
 
-    This automates certain niceties, such as timestamp
-    conversion and serialization.
-    '''
-
-    __dense = False
-
-    def __init__(self, data=None, index=None, columns=None, dtype=None):
-        '''Construct a new JamsFrame object.
-
-        Parameters
-        ----------
-        data
-            Optional data for the new JamsFrame, in any format supported
-            by `pandas.DataFrame.__init__`.
 
-            Fields must be `['time', 'duration', 'value', 'confidence']`.
+class AnnotationData(object):
```

Comment:
honest question: does inheriting directly from `SortedListWithKey` buy us anything? This object owning an `obs` object feels like an unnecessary abstraction, but I'm not sure if this would result in a weird interface.

thoughts?